src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/PowCoinviewRule.cs

```diff
@@ -117,8 +87,11 @@ public virtual void ExecuteBlock(RuleContext context, TaskScheduler taskSchedule
                     // * p2sh (when P2SH enabled in flags and excludes coinbase),
                     // * witness (when witness enabled in flags and excludes coinbase).
                     sigOpsCost += this.GetTransactionSignatureOperationCost(tx, view, flags);
-                    if (sigOpsCost > this.ConsensusOptions.MaxBlockSigopsCost)
+                    if (sigOpsCost > this.consensusOptions.MaxBlockSigopsCost)
+                    {
+                        this.Logger.LogTrace("(-)[BAD_BLOCK_SIG_OPS]");
                         ConsensusErrors.BadBlockSigOps.Throw();
+                    }
 
                     // TODO: Simplify this condition.
                     if (!tx.IsCoinBase && (!context.IsPoS || (context.IsPoS && !tx.IsCoinStake)))
```

Comment:
see issue https://github.com/stratisproject/StratisBitcoinFullNode/issues/1433 for the improvement suggestion